docs/Configuration.md

```diff
@@ -25,6 +25,18 @@ module.exports = {
 
 Please keep in mind that the resulting configuration must be JSON-serializable.
 
+In either JSON or JavaScript, if you'd like to add comments, use a `"//"` key and include your comment as a string value.
+
+```json
+{
+  "name": "my-project",
+  "jest": {
+    "//": "Comment goes here",
+    "verbose": true
+  }
+}
```

Comment:
I don't think we need to document this, but I won't complain

Reply:
So leave as is?

Reply:
Delete here and leave the one below